ReText/window.py

```diff
@@ -1166,7 +1166,8 @@ def aboutDialog(self):
 		+'</a> | <a href="http://daringfireball.net/projects/markdown/syntax">'
 		+self.tr('Markdown syntax')
 		+'</a> | <a href="http://docutils.sourceforge.net/docs/user/rst/quickref.html">'
-		+self.tr('reStructuredText syntax')+'</a></p>')
+		+self.tr('reStructuredText syntax')+'</a></p><p>'
+		+self.tr('Using configuration file at :')+'<br/>'+getSettingsFilePath()+'</p>')
```

Comment:
No space before `:` please, it should be there only in French :)

Reply:
fixed